go/tasks/pluginmachinery/flytek8s/copilot.go

```diff
@@ -43,7 +43,7 @@ func FlyteCoPilotContainer(name string, cfg config.FlyteCoPilotConfig, args []st
 	return v1.Container{
 		Name:       cfg.NamePrefix + name,
 		Image:      cfg.Image,
-		Command:    []string{"/bin/flyte-copilot", "--config", "/etc/flyte/config**/*"},
+		Command:    CopilotCommandArgs(storage.GetConfig()),
```

Comment:
looks good to me, can we update the default copilot image in flyteplugins. [This one](https://github.com/flyteorg/flyteplugins/blob/master/go/tasks/pluginmachinery/flytek8s/config/config.go#L30) should match the one that contains the latest flytestdlib -

Reply:
lgtm after taht